plumpy/communications.py

```diff
@@ -62,13 +62,30 @@ def convert_to_comm(callback: 'Subscriber',
     on the given even loop and return a kiwi future representing the future outcome
     of the original method.
 
-    :param loop: the even loop to schedule the callback in
     :param callback: the function to convert
+    :param loop: the even loop to schedule the callback in
     :return: a new callback function that returns a future
     """
+    if isinstance(callback, kiwipy.BroadcastFilter):
+
+        def _passthrough(*args: Any, **kwargs: Any) -> bool:
+            sender = kwargs.get('sender', args[1])
```

Comment:
To me, the `args` of the subscriber are not very clear, have to check the order of arguments of the original definition every time. Would it be more clear to show the default `None`?
```suggestion
            sender = kwargs.get('sender', None)
```

Reply:
The args are detailed juts st above in the `BroadcastSubscriber` type.
This is not possible because you would have to enforce people always used key-word arguments for the 3 `send` methods (back-breaking), alternatively you would need to use a separate `convert_to_comm` method for broadcast and rpc/task

Reply:
I see, thanks @chrisjsewell